ably/http/http.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, method='GET', url='/', headers=None, body=None, skip_auth=Fal
         self.set_version_headers()
 
     def set_version_headers(self):
+        # RSC7a, RSC7b
```

Comment:
RSC7b specifies an optional variant can be configured, yet this is not included?  See https://github.com/ably/ably-php/blob/de6aa54c154459bc21d64a4e2e1cb20f32acbddd/src/AblyRest.php#L240-L247 for the PHP implementation and https://github.com/ably/ably-ruby/blob/205a4de9add9ee8f6b317e74138aee2b61c6203f/lib/ably/version.rb#L9-L11 for the Ruby implementation